domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3326,6 +3347,26 @@ public Future<PgConnection> getConnection() {
         .map((PgConnection) sqlConnection));
   }
 
+  /**
+   * Get vertx-pg-client connection
+   *
+   * @see #withConn(Function)
+   * @see #withConnection(Function)
+   * @see #withTrans(Function)
+   * @see #withTransaction(Function)
```

Comment:
There are many methods that lists all with* methods in their javadoc. These lists should be extended by the withRead* methods.